### DISCUSSION TITLE
JPEG option for scraper

### DISCUSSION THREAD

**Discussion by oherau** [OnionUI/Onion#1586] [Ideas]
Hi

Covers/box arts take a lot of space with the PNG format (lossless format)
For now the scraper shrink img size to 250x360 max box saving some space on the disk.
But they are still taking a lot of place compared to some games.
Would it be possible to add an option to convert them as jpeg with a reasonable default quality setting (~80%)?

**Comment by Aemiii91** [comment]
The frontend (MainUI) doesn't support JPG, so there's really no point 🤷‍♂️

**Reply by oherau** [reply]
Too bad. If no support is planned for JPG on the long term you can close this topic.

**Reply by Aemiii91** [reply]
Well, we don't control MainUI in that way. But we do plan on writing our own frontend at some point 🤷‍♂️

**Reply by oherau** [reply]
Decoding PNG for display is usually faster and works better on 'pixel' retro designs.
JPEG take much less space and works better with 'real' images.

It would be a nice to preserve storage space, but PNG does the job rather well for now.

**Reply by Aemiii91** [reply]
If you want, you *can* compress PNGs as well (e.g. [online service](https://compresspng.com/))

**Reply by oherau** [reply]
At least I hope scraper uses compression. 
If not it is a point to improve for sure.

**Reply by Nighted** [reply]
Use PngOptimizer. It will free up some space without affecting image quality.
